apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.csproj

```diff
@@ -23,12 +23,12 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="ConfigureAwaitChecker.Analyzer" Version="1.0.1" PrivateAssets="All" />
-    <PackageReference Include="Google.Api.Gax.Grpc" Version="2.6.0" />
+    <PackageReference Include="Google.Api.Gax.Grpc" Version="2.7.0" />
     <PackageReference Include="Google.Cloud.DevTools.Common" Version="1.0.0" />
     <PackageReference Include="Google.Cloud.Logging.V2" Version="2.2.0" />
-    <PackageReference Include="Grpc.Core" Version="1.18.0" PrivateAssets="None" />
+    <PackageReference Include="Grpc.Core" Version="1.21.0" PrivateAssets="None" />
     <PackageReference Include="Microsoft.DotNet.Analyzers.Compatibility" Version="0.2.12-alpha" PrivateAssets="All" />
-    <PackageReference Include="NLog" Version="4.5.11" />
+    <PackageReference Include="NLog" Version="4.6.4" />
```

Comment:
Do we definitely want to do this? That means users of NLog 4.5 will need to upgrade that as well.

Reply:
For optimal user-experience, then I suggest leaving it at 4.5.11 until using NLog 4.6 features (Ex. `AsyncTaskTarget`)

Reply:
OK, put back to 4.5.11